hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/operations/AuthenticationFailureOp.java

```diff
@@ -21,7 +21,10 @@
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.spi.impl.NodeEngineImpl;
 
-public class AuthenticationFailureOperation extends AbstractClusterOperation {
+public class AuthenticationFailureOp extends AbstractClusterOperation {
+
+    public AuthenticationFailureOp() {
```

Comment:
Overriding the default `Operation` constructor will not set the validate target & 64-bit call timeout flags for this operation (https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/spi/Operation.java#L81-L82) -- is this really desirable?